operator/pkg/manifest/installer.go

```diff
@@ -115,7 +127,7 @@ var (
 	}
 
 	installTree      = make(componentTree)
-	dependencyWaitCh = make(map[name.ComponentName]chan struct{})
+	dependencyWaitCh = make(map[name.ComponentName]map[name.ComponentName]chan struct{})
```

Comment:
following from the question above, if dependency on base is transitive through pilot, why do we need a second level of map here? for any given component, there should only ever be one parent to wait for.

Reply:
ack, ill try that out